tests/store/test_store_api.py

```diff
@@ -223,13 +223,13 @@ def test_update_tower_jobs(analysis_store: MockStore, tower_jobs: List[dict], ca
     assert not analysis.failed_jobs
 
     # WHEN jobs are updated
-    analysis_store.update_jobs(analysis=analysis, jobs=tower_jobs)
+    analysis_store.delete_analysis_jobs(analysis=analysis)
 
     # THEN analysis object should have failed jobs
-    assert len(analysis.failed_jobs) == 3
+    assert len(analysis.failed_jobs) == 0
 
     # WHEN jobs are updated a second time
```

Comment:
To be consistent with the previous statement
```suggestion
    # WHEN jobs are updated
```

Reply:
Fixed